modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/ObservationProgressDisplay.scala

```diff
@@ -89,15 +90,22 @@ object SmoothProgressBar {
         else if (remaining > 0) s"${p.fileId} - $durationStr left"
         else s"${p.fileId} - Reading out..."
 
-      Progress(Progress.Props(
-        label       = label,
-        total       = p.total,
-        value       = s.value,
-        color       = "blue".some,
-        progressCls = List(SeqexecStyles.observationProgressBar),
-        barCls      = List(SeqexecStyles.observationBar),
-        labelCls    = List(SeqexecStyles.observationLabel)
-      ))
+      if(p.hideBar)
+        <.div(SeqexecStyles.specialStateLabel,
+
```

Comment:
extra lines?

Reply:
Good catch, thank you.

(I do that sometimes with styles when I'm trying things out and leave the extra lines to remember to move the style to a class before committing, but this one slipped by.)